traffic_ops/v4-client/cdn.go

```diff
@@ -35,73 +36,49 @@ func (to *Session) CreateCDN(cdn tc.CDN) (tc.Alerts, toclientlib.ReqInf, error)
 	return alerts, reqInf, err
 }
 
-func (to *Session) UpdateCDNByIDWithHdr(id int, cdn tc.CDN, header http.Header) (tc.Alerts, toclientlib.ReqInf, error) {
+// UpdateCDNByID replaces the identified CDN with the provided CDN.
+func (to *Session) UpdateCDNByID(id int, cdn tc.CDN, header http.Header) (tc.Alerts, toclientlib.ReqInf, error) {
```

Comment:
I think I've made this comment before, but this could just be `UpdateCDN`

Reply:
You did